handbook/marketing/webinars.md

```diff
@@ -45,6 +45,26 @@ Create the webinar, configuring the following:
 - Registration landing page form
 - Thanks for registering email which includes the webinar link
 
+### Announcement on customer's Sourcegraph instance
+
+Ask the customer:
+
+> Could you add the following configuration to your Sourcegraph instance's global user settings found at https://sourcegraph.example.com/site-admin/global-settings ? This will let all the users visiting Sourcegraph know that they can join the webinar via a dismissible notice:
```

Comment:
```suggestion
> Could you add the following configuration to your Sourcegraph instance's global user settings found at https://sourcegraph.example.com/site-admin/global-settings? This will let all the users visiting Sourcegraph know that they can join the webinar via a dismissible notice:
```